ravedude/src/main.rs

```diff
@@ -92,44 +101,105 @@ fn main() {
 }
 
 fn ravedude() -> anyhow::Result<()> {
-    let args: Args = structopt::StructOpt::from_args();
+    let mut args: Args = structopt::StructOpt::from_args();
+
+    let manifest_path = 'manifest_path: {
+        // By default Cargo scans the current dir and all of its parents for Cargo.toml,
+        // so we mirror its behavior here.
+        let current_dir = std::env::current_dir()?;
+
+        for dir_to_test in current_dir.ancestors() {
+            let path_to_test = dir_to_test.join(Path::new("Ravedude.toml"));
+            if path_to_test.exists() {
+                break 'manifest_path Some(path_to_test);
+            }
+        }
+
+        None
+    };
```

Comment:
I wonder if we should still honor `CARGO_MANIFEST_DIR` when it is present in the environment.  The reason I have in mind is usecases when calling `cargo run` with `--manifest-path` while the working directory is completely different.

Reply:
Good point, done :+1: